std_msgs/Header header
racer/TrajectoryStateMsg[] trajectory
